### PR TITLE
README.md: clean up

### PR DESCRIPTION
**Pull request by xiang90**
Make `Horizontal scalability` section better!

### CODE REVIEW

**Comment by xiang90** [comment]
We need to stop the CI from complaining...

File: README.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ TiKV is a Distributed Key-Value Database which mainly refers to the design of Go
...
 
 - __Horizontal scalability__  
-The Raft and placement driver determine horizontal scalability in TiKV. More specifically，because of reigion as a base unit of data movement in TiKV, similar to spanner's directory, security of membership changing in raft group determines indirectly horizontal scalability. Then the raft implements reigion migration automatically. In addition，scalability is also limited by placement driver. Assuming region's limit storage up to 64MB，only 500 byte needs to be managed in pd. We can see that TiKV may manage data at least in hundred TB level.
+By carefully designing Raft groups and placement driver, we enables horizontal scalability in TiKV. A region, similar to spanner’s directory, is the base unit of data movement in TiKV. A region is controlled by one Raft group and its size is limited to tens MBs . To achieve horizontal scalability, a region can split into two regions when the data size exceeds the limit automatically. Placement driver keeps the metadata of all regions. To ensure horizontal scalability, we ensure the metadata of each region is only 100s of bytes. Together with other techniques, TiKV can easily scale up to hold 100s of TBs of data.
```
Comment (ngaut) [inline README.md]: s/ten/64/

**Comment by ngaut** [comment]
Thank you @xiang90 We are gonna fix the ci issue. Currently we can't reproduce it in our local laptop.

**Comment by xiang90** [comment]
@ngaut Updated... Probably try to use the merge button to squash two commits into one? Or I can squash later (I was using github web to modify the readme)

**Comment by ngaut** [comment]
It's fine, We will use the merge button to squash the commits :)

**Comment by ngaut** [comment]
LGTM, Thank you very much @xiang90

**Pull request merged**